Tests/SwiftSyntaxBuilderTest/IdentifierPatternTests.swift

```diff
@@ -0,0 +1,26 @@
+import XCTest
+import SwiftSyntax
+import SwiftSyntaxBuilder
+
+final class IdentifierPatternTests: XCTestCase {
+  func testStringLiteral() {
+    let leadingTrivia = Trivia.garbageText("␣")
+
+    let testCases: [UInt: (IdentifierPattern, String)] = [
+        #line: (IdentifierPattern(identifier: .identifier("Test")), "␣Test"),
```

Comment:
Since you’re already fixing indentation: This line should be indented two spaces less 😉

Reply:
Woops..